### ISSUE TITLE
[softmax] 算子调研

### ISSUE THREAD

**Issue by bitzyz** [InfiniTensor/operator-design-collection#3]
- framework
  - pytorch
  - paddle
  - onnx
- hardware
  - nvidia
    - cublas
    - cudnn
  - cambricon
  - kunlunxin
